templates/docs/patterns/quote-wrapper/index.md

```diff
@@ -0,0 +1,70 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Quote Wrapper | Patterns
+---
+
+<div class="p-notification--information">
```

Comment:
These docs pages are rendered with [mistune](https://mistune.lepture.com/en/latest/guide.html), not Jinja - so we'd need to research & implement some new way to do this. So, it may be a bit out of scope of this PR, but I do agree this is cumbersome

Reply:
@pastelcyborg I found a way to do this, and I've implemented it. Let me know what you think.
It turns out this can be done with Jinja just like any other template.